2025-07-18 21:31:49,265 INFO    MainThread:29456 [wandb_setup.py:_flush():80] Current SDK version is 0.21.0
2025-07-18 21:31:49,266 INFO    MainThread:29456 [wandb_setup.py:_flush():80] Configure stats pid to 29456
2025-07-18 21:31:49,266 INFO    MainThread:29456 [wandb_setup.py:_flush():80] Loading settings from /Users/davide/.config/wandb/settings
2025-07-18 21:31:49,266 INFO    MainThread:29456 [wandb_setup.py:_flush():80] Loading settings from /Users/davide/Desktop/Legged-Gym-Genesis/genesis_lr/wandb/settings
2025-07-18 21:31:49,266 INFO    MainThread:29456 [wandb_setup.py:_flush():80] Loading settings from environment variables
2025-07-18 21:31:49,267 INFO    MainThread:29456 [wandb_init.py:setup_run_log_directory():703] Logging user logs to /Users/davide/Desktop/Legged-Gym-Genesis/genesis_lr/logs/go2_backflip/Jul18_21-31-40_/wandb/run-20250718_213149-q4v7ipxh/logs/debug.log
2025-07-18 21:31:49,267 INFO    MainThread:29456 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to /Users/davide/Desktop/Legged-Gym-Genesis/genesis_lr/logs/go2_backflip/Jul18_21-31-40_/wandb/run-20250718_213149-q4v7ipxh/logs/debug-internal.log
2025-07-18 21:31:49,267 INFO    MainThread:29456 [wandb_init.py:init():830] calling init triggers
2025-07-18 21:31:49,267 INFO    MainThread:29456 [wandb_init.py:init():835] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-07-18 21:31:49,267 INFO    MainThread:29456 [wandb_init.py:init():871] starting backend
2025-07-18 21:31:49,517 INFO    MainThread:29456 [wandb_init.py:init():874] sending inform_init request
2025-07-18 21:31:49,549 INFO    MainThread:29456 [wandb_init.py:init():882] backend started and connected
2025-07-18 21:31:49,555 INFO    MainThread:29456 [wandb_init.py:init():953] updated telemetry
2025-07-18 21:31:49,591 INFO    MainThread:29456 [wandb_init.py:init():977] communicating run to backend with 90.0 second timeout
2025-07-18 21:31:50,148 INFO    MainThread:29456 [wandb_init.py:init():1029] starting run threads in backend
2025-07-18 21:31:50,231 INFO    MainThread:29456 [wandb_run.py:_console_start():2458] atexit reg
2025-07-18 21:31:50,231 INFO    MainThread:29456 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-07-18 21:31:50,232 INFO    MainThread:29456 [wandb_run.py:_redirect():2375] Wrapping output streams.
2025-07-18 21:31:50,232 INFO    MainThread:29456 [wandb_run.py:_redirect():2398] Redirects installed.
2025-07-18 21:31:50,234 INFO    MainThread:29456 [wandb_init.py:init():1075] run started, returning control to user process
2025-07-18 21:32:27,574 INFO    MsgRouterThr:29456 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
2025-07-18 21:32:30,916 ERROR   MainThread:29456 [redirect.py:_on_write():664] [all runs] error in stdout callback
Traceback (most recent call last):
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/lib/redirect.py", line 662, in _on_write
    cb(written_data)
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 2379, in <lambda>
    lambda data: self._console_raw_callback("stdout", data),
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 398, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 456, in wrapper_fn
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 1566, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/interface/interface.py", line 771, in publish_output_raw
    self._publish_output_raw(o)
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/interface/interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/interface/interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 170, in send_record_publish
    self.send_server_request(server_req)
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 150, in send_server_request
    self._send_message(msg)
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 147, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/opt/miniconda3/envs/gym/lib/python3.11/site-packages/wandb/sdk/lib/sock_client.py", line 126, in _sendall_with_error_handle
    sent = self._sock.send(data)
           ^^^^^^^^^^^^^^^^^^^^^
BrokenPipeError: [Errno 32] Broken pipe
